tiles/src/main/java/com/protomaps/basemap/layers/Boundaries.java

```diff
@@ -21,29 +21,213 @@ public String name() {
+        }
+        case "Line of control (please verify)" -> {
+          kind = "country";
+          kind_detail = "line_of_control";
+          disputed = true;
+        }
+        case "Overlay limit" -> {
+          kind = "overlay_limit";
+          admin_level = 3;
+        }
+        case "Unrecognized" -> kind = "unrecognized_country";
+        case "Map unit boundary" -> {
+          kind = "map_unit";
+          admin_level = 3;
+        }
+        case "Breakaway" -> {
+          kind = "unrecognized_country";
+          kind_detail = "disputed_breakaway";
+          admin_level = 3;
+        }
+        case "Claim boundary" -> {
+          kind = "unrecognized_country";
+          kind_detail = "disputed_claim";
+          admin_level = 3;
+        }
+        case "Elusive frontier" -> {
+          kind = "unrecognized_country";
+          kind_detail = "disputed_elusive";
+          admin_level = 3;
+        }
+        case "Reference line" -> {
+          kind = "unrecognized_country";
+          kind_detail = "disputed_reference_line";
+          admin_level = 3;
+        }
+        case "Admin-1 region boundary" -> {
+          kind = "macroregion";
+          admin_level = 3;
+        }
+        case "Admin-1 boundary" -> {
+          kind = "region";
+          admin_level = 4;
+        }
+        case "Admin-1 statistical boundary" -> {
+          kind = "region";
+          admin_level = 4;
+        }
+        case "Admin-1 statistical meta bounds" -> {
+          kind = "region";
+          admin_level = 4;
+        }
+        case "1st Order Admin Lines" -> {
+          kind = "region";
+          admin_level = 4;
+        }
+        case "Unrecognized Admin-1 region boundary" -> {
+          kind = "unrecognized_macroregion";
+          admin_level = 4;
+        }
+        case "Unrecognized Admin-1 boundary" -> {
+          kind = "unrecognized_region";
+          admin_level = 4;
+        }
+        case "Unrecognized Admin-1 statistical boundary" -> {
+          kind = "unrecognized_region";
+          admin_level = 4;
+        }
+        case "Unrecognized Admin-1 statistical meta bounds" -> {
+          kind = "unrecognized_region";
+          admin_level = 4;
+        }
+        default -> kind = "";
+      }
+    }
+
+    if (sf.canBeLine() && sf.hasTag("min_zoom") && (kind.equals("") == false && kind.equals("tz_boundary") == false)) {
+      features.line(this.name())
+        // Don't label lines to reduce file size (and they aren't shown in styles anyhow)
+        //.setAttr("name", sf.getString("name"))
+        .setAttr("pmap:min_zoom", sf.getLong("min_zoom"))
+        .setAttr("pmap:min_admin_level", admin_level)
+        .setZoomRange(
+          sf.getString("min_zoom") == null ? theme_min_zoom : (int) Double.parseDouble(sf.getString("min_zoom")),
+          theme_max_zoom)
+        .setAttr("pmap:ne_id", sf.getString("ne_id"))
+        .setAttr("pmap:brk_a3", sf.getString("brk_a3"))
```

Comment:
This might be an oversight in the documentation with the final v1.9 changes? It was added, IIRC, because it allows the cartographer the flexibility to turn off/on certain country boundary lines by ID in the style explicitly as a fail safe for legal compliance without needing a data or code change (which can take time).

Reply:
Sounds good to me, but if it overlaps with `ne_id` in use cases I'd prefer the one with more semantics (how often are NE IDs reassigned?)

Reply:
`ne:brk_a3` is highly stable, while the `ne:id` is mostly stable but may vary between scale set (1:10m, 1:50m, 1:110m) and major Natural Earth release versions.